examples/leasing/node_release_script.py

```diff
@@ -0,0 +1,212 @@
+#! /bin/python
+"""
+This script will free nodes from a project after
+a specific amount of time.
+Script reads a config file from /etc/leasing.cfg
+which defines which nodes are in the project, time threshold
+admin user name and password and status file.
+
+status file includes each node's status, which node is in
+the project, which one is free and for how long the node
+has been in the project.
+
+*******************
+
+Config file format (/etc/leasing.cfg):
+node_list: names of the nodes
+threshold: number of times that cron job is called before
+            releasing the node
+user_name:
+password:
+endpoint: the ip address and the port number
+            which hil is running on
+status_file: it should be /var/lib/leasing
+
+******************
```

Comment:
Just leave pointers saying that this script requires a config file and status file, but move the actual description to the config and status file itself. You could follow [this example](https://github.com/CCI-MOC/hil/blob/master/examples/hil.cfg) on how to document those files.

Reply:
I was thinking about having a README file in the folder.

Reply:
that's fine too. But make sure the config/status file descriptions are in those files itself.